src/map/itemdb.cpp

```diff
@@ -1267,15 +1260,17 @@ static void itemdb_pc_get_itemgroup_sub(struct map_session_data *sd, bool identi
 		tmp.card[3] = GetWord(sd->status.char_id, 1);
 	}
 
-	if (!itemdb_isstackable(data->nameid))
-		get_amt = 1;
-	else
+	uint16 get_amt = 0;
+
+	if (itemdb_isstackable(data->nameid) && data->GUID == 0)
```

Comment:
This should actually take the "GUID" parameter of the item db into account as well not only the "GUID" from item group db.